java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/persistence/DynamoDbMetastoreImpl.java

```diff
@@ -28,6 +28,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * Provides an AWS DynamoDB based implementation of {@link Metastore} to store and retrieve
+ * {@link com.godaddy.asherah.appencryption.utils.Json} values for system and intermediate keys. It uses the default
+ * table name "EncryptionKey" but that can be configured using {@link Builder#withTableName(String)} " option.
+ * Creation time is stored in unix time seconds.
```

Comment:
```suggestion
 * Creation time is stored as unix time seconds.
```

Reply:
I think both of them convey the same idea and we've used `in` for the JdbcMetastoreImpl class.